beacon-chain/types/state.go

```diff
@@ -37,6 +37,11 @@ func NewGenesisStates() (*ActiveState, *CrystallizedState) {
 			RecentBlockHashes:   [][]byte{},
 		},
 	}
+	var validators []*pb.ValidatorRecord
+	for i := 0; i < 100; i++ {
+		validator := &pb.ValidatorRecord{StartDynasty: 0, EndDynasty: 100, Balance: 1000, WithdrawalAddress: []byte{}, PublicKey: 0}
```

Comment:
Set `EndDynasty` to `params.DefaultEndDynasty` or we'll run into issues with simulate block advancing and dynasty passes 100 mark

Reply:
Also `balance` to `params.DefaultBalance`